index.html

```diff
@@ -5,55 +5,55 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Mirco Warnck</title>
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="./styles/main.css" />
   </head>
 
   <body>
-    <header>
+    <header class="header">
       <h1>Mirco Warnck</h1>
-      <nav>
-        <ul>
+      <nav class="nav">
+        <ul class="nav__ul">
           <li>
-            <a class="nav-links" href="#biography">Biography</a>
+            <a class="nav__nav-links" href="#biography">Biography</a>
           </li>
           <li>
-            <a class="nav-links" href="#contact">Contact</a>
+            <a class="nav__nav-links" href="#contact">Contact</a>
           </li>
           <li>
-            <a class="nav-links" href="#networks">Networks</a>
+            <a class="nav__nav-links" href="#networks">Networks</a>
           </li>
         </ul>
       </nav>
     </header>
     <main>
-      <section id="image-skills" class="wrapper">
-        <figure class="illustration">
+      <section class="image-skills wrapper" id="image-skills">
+        <figure class="image-skills__illustration">
           <img
-            class="header-img"
+            class="image-skills__header-img"
             src="https://source.unsplash.com/random/400x300"
             alt=""
           />
           <figcaption>A random image for illustration</figcaption>
         </figure>
-        <div class="skills">
+        <div>
           <h2>Short facts</h2>
-          <ul class="skills-trans">
+          <ul class="image-skills__skills-trans">
             <li>born in Lübeck</li>
             <li>Bachelor of Science</li>
             <li>Web Developer</li>
             <li>Sports (Running, Fitness)</li>
           </ul>
           <h2>Skills</h2>
-          <ul class="skills-trans">
+          <ul class="image-skills__skills-trans">
```

Comment:
here for eg u could do it like in the nav section `image-skills__ul` or `image-skills__list`